docs/queries/queries.md

```diff
@@ -397,30 +427,30 @@ You can also apply filters when needed.
 
 ### Exists
 
-Returns a boolean confirming if a specific record exists.
+Returns a boolean confirming if a specific record exists. Use cache if possible.
 
 ```python
 exists = await User.query.filter(email="foo@bar.com").exists()
 ```
 
-### Count
-
-Returns an integer with the total of records.
-
-```python
-total = await User.query.count()
-```
-
 ### Contains
 
-Returns true if the QuerySet contains the provided object.
+Returns true if the QuerySet contains the provided object. Use cache if possible.
 
 ```python
 user = await User.query.create(email="foo@bar.com")
 
 exists = await User.query.contains(instance=user)
 ```
 
+
+### Count
+
+Returns an integer with the total of records. Cached.
```

Comment:
Again, if it's cached or not, that implementation. You can instead of changing the existing texts, add a note at the bottom of the document and explain all of that in a simple language. Again, keep it simple.